src/README.md

```diff
@@ -242,8 +242,8 @@ file inside your source code repository.
 
     Set this variable to a non-empty value to have the 'pip' program and related
     python packages (setuptools and wheel) be upgraded to the most recent version
-    before any Python packages are installed. If not set it will use whatever
-    the default version is included by the platform for the Python version being used.
+    before any Python packages are installed. If not set it will use the default
+    version from the latest stable Fedora we have in the source repository in a form of wheel.
```

Comment:
```suggestion
    before any Python packages are installed. If not set, the container will use
    the stable pip version this container was built with, taken from a recent Fedora release.
```

If we plan to update once every year, it will not always be from the latest Fedora. What do you think of this wording?